usage-examples/csharp/driver/Examples/Aggregation/GroupTotal.cs

```diff
@@ -0,0 +1,134 @@
+//	:replace-start: {
+//	  "terms": {
+//	    "_order": "order",
+//      "_aggDb": "aggDb"
+//	  }
+//	}
+namespace Examples.Aggregation;
+
+using MongoDB.Driver;
+using MongoDB.Bson;
+
+public class GroupTotal
+{
+    private IMongoDatabase _aggDB;
+    private IMongoCollection<Order> _orders;
+    
+    public void SeedData()
+    {
+        DotNetEnv.Env.TraversePath().Load();
+        string uri = DotNetEnv.Env.GetString("CONNECTION_STRING", "Env variable not found. Verify you have a .env file with a valid connection string.");
```

Comment:
usually use `var` instead of `string` unless it's not clear what is being returned.